bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```
@@ -568,6 +568,23 @@ CheckedTreeSelectionDialog_nothing_available=No entries available.
 CheckedTreeSelectionDialog_select_all=Select &All
 CheckedTreeSelectionDialog_deselect_all=&Deselect All
 
+WindowsDefenderConfigurator_statusCheck=Windows Defender Exclusion Check
+WindowsDefenderConfigurator_exclusionCheckMessage=Microsoft's Windows Defender is active and could significantly decrease the application's start-up and overall performance.\n\
+Select how this installation should be handled by Windows Defender:
+WindowsDefenderConfigurator_exclusionInformation=If Windows Defender is active and scans this application it can significantly slow down its start-up and overall performance.\n\
+To prevent a decrease in performance {0} can exclude itself and all files opened from real-time scanning by Windows Defender.\n\
+Be aware that in general adding exclusions could affect this computer's security and adding them requires Administrator privileges.
+WindowsDefenderConfigurator_scriptButtonLabel=Powershell script (execution requires Administrator privileges)
+WindowsDefenderConfigurator_performExclusionChoice=Exclude {0} from being scanned to improve performance.\n\
+(In general adding exclusions may affect the security level of this computer)
+WindowsDefenderConfigurator_ignoreThisInstallationChoice=Keep {0} being scanned by Windows Defender
+WindowsDefenderConfigurator_ignoreAllChoice=Skip startup check for all new Eclipse based installations
+WindowsDefenderConfigurator_detailsAndOptionsLinkText=See '<a>Startup and Shutdown</a>' preference for more details and configuration options.
+WindowsDefenderConfigurator_runExclusionFromPreferenceButtonLabel=Run exclusion check now
+WindowsDefenderConfigurator_statusInactive=Windows Defender is not active on this computer.
+WindowsDefenderConfigurator_statusCheckFailed=Failed to retrieve Windows Defender status
```

Maybe this should have a "." at the end:
```suggestion
WindowsDefenderConfigurator_statusCheckFailed=Failed to retrieve Windows Defender status.
```

---

Yes, added a dot.